iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -433,6 +454,14 @@ TEST_F(PoshRuntime_test, CreateNodeReturnValue)
     // EXPECT_EQ(nodeDeviceIdentifier, nodeData->m_nodeDeviceIdentifier);
 }
 
+TEST_F(PoshRuntime_test, CreateNodeError)
```

Comment:
Maybe name it `CreatingNodeWithInvalidNameLeadsToTermination`

Reply:
@elfenpiff As @JimmyBch already stated the test fails to die if build in release mode. How do we proceed with this test? Remove it, disable it or ...?
Probably the same problem with ``GetMiddlewareInterfaceIsNotSuccessful``.

Reply:
@JimmyBch if i see it right, these are both function which call the errorhandler, therefore you need to test if the errorhandler is called with the right errorcode.
Please take a look at: https://github.com/eclipse-iceoryx/iceoryx/pull/463#discussion_r553852806
Maybe this helps.

Reply:
Thanks for the hint :)